pyre/extra_tests/snippets/multiprocessing_semlock_keywords.py

```
@@ -0,0 +1,39 @@
+
+RECURSIVE_MUTEX = 0
+base = f'/pyre-semlock-{os.getpid()}'
+
+by_keyword = _multiprocessing.SemLock(
+    RECURSIVE_MUTEX, 1, 1, name=f'{base}-kw', unlink=True
+)
+assert by_keyword.kind == RECURSIVE_MUTEX
+assert by_keyword.maxvalue == 1
+
+by_position = _multiprocessing.SemLock(
+    RECURSIVE_MUTEX, 1, 1, f'{base}-pos', True
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)